js/heroes-talents.js

```diff
@@ -162,10 +162,6 @@ class HeroesTalents {
     return this._roles[role].all.length;
   }
 
-  get heroCount() {
```

Comment:
heroCount is used, don't remove. see https://github.com/ebshimizu/stats-of-the-storm/blob/master/js/hero-collection.js#L257

Reply:
was there twice :)